src/ParameterDeriverTrait.php

```diff
@@ -111,4 +114,13 @@ protected function isClassCallable($callable) : bool
     {
         return (is_array($callable) && is_string($callable[0]) && class_exists($callable[0]));
     }
+
+    /**
+     * @param callable $callable
+     * @return True if the callable represents an invokeable object, false otherwise.
```

Comment:
Honestly, these annotations aren't necessary, as they give no additional information beyond what the signature provides already. IDEs and phpdoc both pick up the typehints from the signature.

Reply:
That is in line with how other similar methods are documented.

Reply:
If nothing else, fix the spelling: `s/invokeable/invokable/`. I'll defer to @Crell on whether or not to keep the docblock.

Reply:
For consistency the docblock should remain, and have a description line.  Add that and I can merge.